pkg/legacy/importer.go

```diff
@@ -29,7 +29,6 @@ var (
 		"forwarder_timeout",
 		"default_integration_http_timeout",
 		"collect_ec2_tags",
```

Comment:
I realize now that we haven't documented the `collect_ec2_tags` option in https://github.com/DataDog/datadog-agent/blob/master/pkg/config/config_template.yaml, could you add it there?